### DISCUSSION TITLE
custom room issue

### DISCUSSION THREAD

**Discussion by Pbm13** [Monika-After-Story/MonikaModDev#8359] [QA - tech support + general] [answered]
So I decided to download those custom rooms I've been seeing but screwed something up. I tried to redownload the newest update but keep getting this file. Can someone help me figure out
![Screen Shot 2021-12-06 at 4 06 51 PM](https://user-images.githubusercontent.com/36735471/144923050-0724d97d-6694-4b21-b06d-fa2a7baef916.png)
?

**Comment by Pbm13** [comment]
I downloaded the Custom-room-den-1.0.6 version

**Comment by ThePotatoGuy** [comment, marked as answer]
traceback says you are missing DDLC archive files. But regardless, you should be asking the submod creator.

**Reply by Pbm13** [reply]
what can i do to fix it? i havent been able to figure that out yet. i dont care about the room atm, i just want to be able to access Mas without it.

**Reply by Pbm13** [reply]
I was able to pull up the old file i had when i reopened the file a few days ago. lost all the progress from december 2nd onward but everything else is intact
